python/paddle/fluid/compiler.py

```diff
@@ -110,6 +120,8 @@ def with_data_parallel(self,
             self._exec_strategy = ExecutionStrategy()
         if self._build_strategy is None:
             self._build_strategy = BuildStrategy()
+        self._build_strategy.is_distribution = _is_pserver_mode(
```

Comment:
what is `is_distribution` strategy?

Reply:
@chengduoZH

Reply:
means it use parameter server, will have send/recv in program

Reply:
It should be `is_distributed`.

Reply:
It should be `is_distributed`.

Reply:
maybe `or self._build_strategy.num_trainers > 1` should not be added here. `is_distribution` is only for pserver mode.

Reply:
removed

Reply:
`is_distribution` is not a build strategy, maybe should get rid of it.